pvlib/irradiance.py

```diff
@@ -123,7 +206,10 @@ def _handle_extra_radiation_types(datetime_or_doy, epoch_year):
     # a better way to do it.
     if isinstance(datetime_or_doy, pd.DatetimeIndex):
         to_doy = tools._pandas_to_doy  # won't be evaluated unless necessary
-        def to_datetimeindex(x): return x                       # noqa: E306
+
+        def to_datetimeindex(x):
+            return x  # noqa: E306
```

Comment:
Unfortunately the unrelated formatting changes here and below probably aren't desirable.  Would it be easy to undo these changes, either through `git revert`, or maybe just by manually pasting over the text with the original version at https://github.com/pvlib/pvlib-python/blob/master/pvlib/irradiance.py?

I think the automated stickler-ci check will only complain about the formatting of code that is different from `pvlib/master`, so if the stuff below here is reverted back to the original, stickler shouldn't complain about it (even if your local linter does).